packages/aws-cdk-lib/aws-cloudfront/test/origin-access-control.test.ts

```diff
@@ -59,4 +59,56 @@ describe('S3OriginAccessControl', () => {
 
     expect(imported.originAccessControlId).toEqual(originAccessControlId);
   });
+
+  test('creates a FunctionUrlOriginAccessControl with default properties', () => {
```

Comment:
Can we nest the new tests under a new `describe`, e.g. 
```
describe('FunctionUrlOriginAccessControl', () => {
    test(...);
});
```

Reply:
fixed.
https://github.com/aws/aws-cdk/pull/31339/commits/3731370b9b8f8f6362f6d9b4b983108767f0dff2